frontend/src/Pages/AdminManagement.tsx

```diff
@@ -302,13 +303,18 @@ export default function AdminManagement() {
             <Modal
                 ref={addUserModal}
                 type={ModalType.Add}
-                item="User"
-                form={<AddUserForm onSuccess={onAddUserSuccess} />}
+                item="Administrator"
+                form={
+                    <AddUserForm
+                        onSuccess={onAddUserSuccess}
+                        userRole={UserRole.Admin}
+                    />
+                }
             />
             <Modal
                 ref={editUserModal}
                 type={ModalType.Edit}
-                item="Admin"
+                item="Administrator"
```

Comment:
Stick with Admin (over Administrator) since all other parts of the page such as the titles and buttons use **Admin**